{"uid":"d1d20cbb04034385","name":"test_redeploy_charm_same_model","fullName":"tests.integration.test_charm#test_redeploy_charm_same_model","historyId":"3c325508c183b9f3c1e6bfe3a2df9380","time":{"start":1708651651915,"stop":1708651651915,"duration":0},"description":"Redeploy the charm in the same model to test that it works.","descriptionHtml":"<p>Redeploy the charm in the same model to test that it works.</p>\n","status":"skipped","statusMessage":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/postgresql-k8s-operator/postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 345, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/runner/work/postgresql-k8s-operator/postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 266, in <lambda>\n    lambda: ihook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/postgresql-k8s-operator/postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 493, in __call__\n    return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)\n  File \"/home/runner/work/postgresql-k8s-operator/postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 115, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/postgresql-k8s-operator/postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 152, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/postgresql-k8s-operator/postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 114, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/postgresql-k8s-operator/postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 137, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/postgresql-k8s-operator/postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 85, in pytest_runtest_setup\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/work/postgresql-k8s-operator/postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 65, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/work/postgresql-k8s-operator/postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 137, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/postgresql-k8s-operator/postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 833, in pytest_runtest_setup\n    yield from self._runtest_for(item, \"setup\")\n  File \"/home/runner/work/postgresql-k8s-operator/postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 822, in _runtest_for\n    yield\n  File \"/home/runner/work/postgresql-k8s-operator/postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 137, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/postgresql-k8s-operator/postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 877, in pytest_runtest_setup\n    return (yield)\n  File \"/home/runner/work/postgresql-k8s-operator/postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 137, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/postgresql-k8s-operator/postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 82, in pytest_runtest_setup\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/work/postgresql-k8s-operator/postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 63, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/work/postgresql-k8s-operator/postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/postgresql-k8s-operator/postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 161, in pytest_runtest_setup\n    item.session._setupstate.setup(item)\n  File \"/home/runner/work/postgresql-k8s-operator/postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 517, in setup\n    raise exc\n  File \"/home/runner/work/postgresql-k8s-operator/postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 514, in setup\n    col.setup()\n  File \"/home/runner/work/postgresql-k8s-operator/postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1839, in setup\n    self._request._fillfixtures()\n  File \"/home/runner/work/postgresql-k8s-operator/postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 693, in _fillfixtures\n    item.funcargs[argname] = self.getfixturevalue(argname)\n  File \"/home/runner/work/postgresql-k8s-operator/postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 547, in getfixturevalue\n    fixturedef = self._get_active_fixturedef(argname)\n  File \"/home/runner/work/postgresql-k8s-operator/postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 566, in _get_active_fixturedef\n    self._compute_fixture_value(fixturedef)\n  File \"/home/runner/work/postgresql-k8s-operator/postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 652, in _compute_fixture_value\n    fixturedef.execute(request=subrequest)\n  File \"/home/runner/work/postgresql-k8s-operator/postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1095, in execute\n    result = ihook.pytest_fixture_setup(fixturedef=self, request=request)\n  File \"/home/runner/work/postgresql-k8s-operator/postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 493, in __call__\n    return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)\n  File \"/home/runner/work/postgresql-k8s-operator/postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 115, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/postgresql-k8s-operator/postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 152, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/postgresql-k8s-operator/postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 114, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/postgresql-k8s-operator/postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 137, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/postgresql-k8s-operator/postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/home/runner/work/postgresql-k8s-operator/postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/postgresql-k8s-operator/postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1149, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/work/postgresql-k8s-operator/postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 916, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/runner/work/postgresql-k8s-operator/postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_operator/plugin.py\", line 216, in abort_on_fail\n    pytest.xfail(\"aborted\")\n  File \"/home/runner/work/postgresql-k8s-operator/postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/outcomes.py\", line 248, in xfail\n    raise XFailed(reason)\n_pytest.outcomes.XFailed: aborted","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"event_loop","time":{"start":1708651638032,"stop":1708651638032,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1708651638032,"stop":1708651638033,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1708651638033,"stop":1708651638264,"duration":231},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1708651651915,"stop":1708651651915,"duration":0},"status":"failed","statusMessage":"_pytest.outcomes.XFailed: aborted\n","statusTrace":"  File \"/home/runner/work/postgresql-k8s-operator/postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 137, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/postgresql-k8s-operator/postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/home/runner/work/postgresql-k8s-operator/postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/postgresql-k8s-operator/postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1149, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/work/postgresql-k8s-operator/postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 916, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/runner/work/postgresql-k8s-operator/postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_operator/plugin.py\", line 216, in abort_on_fail\n    pytest.xfail(\"aborted\")\n  File \"/home/runner/work/postgresql-k8s-operator/postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/outcomes.py\", line 248, in xfail\n    raise XFailed(reason)\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"afterStages":[{"name":"ops_test::finalizer","time":{"start":1708651651920,"stop":1708651652283,"duration":363},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1708651652284,"stop":1708651652285,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1708651652285,"stop":1708651652285,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1708651652285,"stop":1708651652285,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"group(1)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration"},{"name":"suite","value":"test_charm"},{"name":"host","value":"fv-az915-266"},{"name":"thread","value":"12495-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.test_charm"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"history":{"statistic":{"failed":0,"broken":0,"skipped":4,"passed":19,"unknown":0,"total":23},"items":[{"uid":"cc3886f43488f70f","reportUrl":"../1919//#testresult/cc3886f43488f70f","status":"passed","time":{"start":1708566765744,"stop":1708567024389,"duration":258645}},{"uid":"c01db9849d0eb044","reportUrl":"../1916//#testresult/c01db9849d0eb044","status":"passed","time":{"start":1708480407043,"stop":1708480674079,"duration":267036}},{"uid":"8ee18152f742dc40","reportUrl":"../1911//#testresult/8ee18152f742dc40","status":"passed","time":{"start":1708393954503,"stop":1708394238888,"duration":284385}},{"uid":"1f97fd962d51785a","reportUrl":"../1908//#testresult/1f97fd962d51785a","status":"passed","time":{"start":1708307712531,"stop":1708307987110,"duration":274579}},{"uid":"dcf878df9d98305","reportUrl":"../1906//#testresult/dcf878df9d98305","status":"passed","time":{"start":1708221413341,"stop":1708221682447,"duration":269106}},{"uid":"7292e10f96a6c492","reportUrl":"../1905//#testresult/7292e10f96a6c492","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1708133260964,"stop":1708133260964,"duration":0}},{"uid":"92cf3472c7daf670","reportUrl":"../1902//#testresult/92cf3472c7daf670","status":"passed","time":{"start":1708048277369,"stop":1708048529558,"duration":252189}},{"uid":"448745411e58ee2e","reportUrl":"../1896//#testresult/448745411e58ee2e","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1707960503900,"stop":1707960503900,"duration":0}},{"uid":"4ee518aee8b58f81","reportUrl":"../1892//#testresult/4ee518aee8b58f81","status":"passed","time":{"start":1707875696440,"stop":1707875960172,"duration":263732}},{"uid":"446d6ba1eb37b777","reportUrl":"../1884//#testresult/446d6ba1eb37b777","status":"passed","time":{"start":1707789228773,"stop":1707789470472,"duration":241699}},{"uid":"7d3526a231eca7ba","reportUrl":"../1881//#testresult/7d3526a231eca7ba","status":"passed","time":{"start":1707702687669,"stop":1707702939417,"duration":251748}},{"uid":"46169ab84924f927","reportUrl":"../1880//#testresult/46169ab84924f927","status":"passed","time":{"start":1707616693256,"stop":1707616960382,"duration":267126}},{"uid":"86dda9017f6586be","reportUrl":"../1879//#testresult/86dda9017f6586be","status":"passed","time":{"start":1707529959953,"stop":1707530200043,"duration":240090}},{"uid":"10bca7683495682b","reportUrl":"../1875//#testresult/10bca7683495682b","status":"passed","time":{"start":1707443562703,"stop":1707443845047,"duration":282344}},{"uid":"b777ca7049606cb4","reportUrl":"../1870//#testresult/b777ca7049606cb4","status":"passed","time":{"start":1707357136503,"stop":1707357416478,"duration":279975}},{"uid":"8082c307a46aff56","reportUrl":"../1866//#testresult/8082c307a46aff56","status":"passed","time":{"start":1707270737134,"stop":1707270989278,"duration":252144}},{"uid":"69346d8117234fef","reportUrl":"../1862//#testresult/69346d8117234fef","status":"passed","time":{"start":1707184420620,"stop":1707184687835,"duration":267215}},{"uid":"117a7fe3be716a30","reportUrl":"../1852//#testresult/117a7fe3be716a30","status":"passed","time":{"start":1707097967563,"stop":1707098216906,"duration":249343}},{"uid":"311a0bedaa61de4b","reportUrl":"../1851//#testresult/311a0bedaa61de4b","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1707010182995,"stop":1707010182995,"duration":0}},{"uid":"52564f764fd959b9","reportUrl":"../1850//#testresult/52564f764fd959b9","status":"passed","time":{"start":1706925055016,"stop":1706925263089,"duration":208073}}]},"tags":["asyncio","group(1)"]},"source":"d1d20cbb04034385.json","parameterValues":[]}